travis/large-requirements.txt

```diff
@@ -28,7 +28,7 @@ typing==3.6.6
 pysftp==0.2.9
 keras==2.3.1
 attrdict==2.0.0
-azureml-sdk==1.0.17; python_version >= "3.0"
+azureml-sdk==1.2.0; python_version >= "3.0"
```

Comment:
Would there be value in making this some kind of ~= dependency?

Reply:
No. AzureML doesn't follow semver (because we're bad), so we need to pin to an exact version.